app/jobs/deny_missed_requests_job.rb

```diff
@@ -0,0 +1,32 @@
+class DenyMissedRequestsJob < ReservationJob
+  include EmailJobHelper
+
+  private
+
+  def log_start
+    missed_requests
+    super(type: 'missed requests', task: 'denying')
+  end
+
+  def missed_requests
+    collection(:missed_requests)
+  end
+
+  def run
+    missed_requests.each do |r|
+      log_denial r
+      deny r
+      send_email r
+    end
+  end
+
+  def deny(res)
+    res.status = 'denied'
```

Comment:
Should we just make this a model method instead? That seems to make more sense; the job shouldn't necessarily know how to deny a request, only that each reservation needs to be sent the message `deny`. What do you think?